circle.yml

```diff
@@ -1066,6 +1066,31 @@ jobs:
       - run:
           name: Release
           command: yarn semantic-release @cypress/vue
+          
+  npm-react:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Build
+          command: yarn workspace @cypress/react build
+      - run:
+          name: Run unit test
+          command: yarn workspace @cypress/react test:unit 
+      - run:
+          name: Run tests
+          command: yarn workspace @cypress/react test
```

Comment:
doest this run our example applications to test it?

Reply:
No, we can discuss it further, but I think that we should not run tests for file-system based examples as a part of CI flow. Because it takes too much time, requires the installation of a giant amount of unrelated dependencies. 

Most of repos containing such examples do not build them on each PR. Check next.js, material-ui. Would be much useful better to include builds for different versions of webpack, experimental version of react, etc.

Reply:
Ok, but you see how in cypress-react-unit-test you can change something and _none_ of the existing projects would break right since we have lots of unit tests and complete application tests (aside from accidental dependency found in the root)? And if we want to test different webpack versions, it is the same thing? 

So this merge skips what the users actually experience, and thus we cannot guarantee that a new version is not breaking something accidentally, so I have to reject this. The example app tests are there for a reason - because they protect our users.

Reply:
The example app tests are redundant in most cases. While they are great examples, the security they provide is often redundant. The differences between Happo, Percy, and Applitools are not relevant to the core repository's functionality.

For example, you don't see lodash testing millions of projects. With a thoughtfully crafted CI strategy, we can retain a comfortable level of security. I would suggest a nightly or bi-weekly build against the example repositories to accomplish this. As we are in alpha with a relatively small userbase and do not promise 100% stability, I am comfortable trading ecosystem coverage for speed.

Lastly, we are not going entirely without coverage. When we change @cypress/react, we trigger a full Cypress binary build and dogfood the changes in an e2e way.